app/helpers/miq_ae_class_helper.rb

```diff
@@ -41,7 +44,7 @@ def class_prefix(cls)
     case cls.to_s.split("::").last
     when "MiqAeClass"
       "aec"
-    when "MiqAeNamespace"
+    when "MiqAeDomain", "MiqAeNamespace"
```

Comment:
What is class prefix used for,  a route? Can we ask cls itself and not specify class names?